deeposlandia/kerastrain.py

```diff
@@ -210,23 +210,56 @@ def add_training_arguments(parser):
     STEPS = args.nb_training_image // args.batch_size
     VAL_STEPS = args.nb_validation_image // args.batch_size
     TEST_STEPS = args.nb_testing_image // args.batch_size
+
+    checkpoint_path = os.path.join(folders["output"], "checkpoints", instance_name)
+    if os.path.isdir(checkpoint_path):
+        checkpoints = os.listdir(checkpoint_path)
+        if len(checkpoints) > 0:
+            model_checkpoint = max(checkpoints)
```

Comment:
This a alphanum order. Will the selected file be always the most recent?

Reply:
As we define the checkpoint with the parameter `save_best_only=True`, and as the checkpoint names depend only on current training epoch, all the saved model are supposed to provide a better validation accuracy than the previous ones. In such a context, considering the most recent checkpoint guarantees recovering the best validation accuracy.